File: 2.Prgramming fundamentals/kurs fundamental 2017/09.DictionariesLambdaAndLINQExercises/01.Phonebook/Phonebook.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace _01.Phonebook
{
    class Phonebook
    {
        static void Main(string[] args)
        {
            List<string> input = Console.ReadLine().Split(' ').ToList();

            Dictionary<string, string> nameAndNumber = new Dictionary<string, string>();
            
            while (input[0] != "END")
            {

                if (input[0].Equals("A"))
                {
                    string name = input[1];
                    string number = input[2];

                    nameAndNumber[name] = number;
                }
                else if (input[0].Equals("S"))
                {
                    string name = input[1];

                    if(nameAndNumber.ContainsKey(name))
                        Console.WriteLine($"{name} -> {nameAndNumber[name]}");
                    else
                        Console.WriteLine($"Contact {name} does not exist.");
                }

                input = Console.ReadLine().Split(' ').ToList();
            }
            
        }
    }
}

File: 3.Software Technologies/Software Technologies 2017/16.JavaSintaxBasicWebExercises/src/Phonebook.java
import java.util.*;

public class Phonebook {
    public static void main(String[] args) {

        Scanner scanner = new Scanner(System.in);
        HashMap<String, String> phoneBook= new HashMap<String, String>();

        String[] input = scanner.nextLine().split("\\s");
       // ArrayList<String> searchedNames = new ArrayList<>();



        while(!input[0].equals("END"))
        {
            String command = input[0];
            if(command.equals("A"))
            {
                String name = input[1];
                String number = input[2];


                phoneBook.put(name,number);

            }
            else if(command.equals("S")) {

                String name = input[1];
                if (phoneBook.containsKey(name)) {
                    String phone = phoneBook.get(name);
                    System.out.println(name + " -> " + phone);
                }
                else
                {
                    System.out.println("Contact " + name + " does not exist.");
                }
            }


            input = scanner.nextLine().split("\\s");

        }

       /* for (String searchedName : searchedNames) {
            if(phoneBook.containsKey(searchedName))
            {
                System.out.println(searchedName + " -> " + phoneBook.get(searchedName));
            }
            else
            {
                System.out.println("Contact " + searchedName + " does not exist.");
            }
        }
        */


    }
}
